app/controllers/OERWorldMap.java

```diff
@@ -235,18 +212,34 @@ public String getLabel(String aId) {
       // Return requested language
       for (Object n : ((ArrayList) name)) {
         if (n instanceof Resource) {
-          String language = ((Resource) n).getAsString("@language");
+          Resource r = (Resource) n;
```

Comment:
Can't we use the `ModelCommon` base class here instead of checking for each Subclass separately?

Reply:
Yes. Next commit.